README.md

```diff
@@ -202,6 +203,43 @@ Where "dir" is a subdirectory within the repo to push all the check files. Setup
 4. Create `updates/.gitkeep`.
 5. Enable GitHub Pages in your settings for your desired branch.
 
+#### SQL(ite) Storage
+
+**[godoc: SQL](https://godoc.org/github.com/sourcegraph/checkup#SQL)**
+
+Postgres or sqlite3 databases can be used as storage backends.
+
+sqlite database file configuration:
```

Comment:
Document how to select SQLite or Postgresql.

Reply:
@mholt (I've added this documentation a little bit further, but github doesn't know that :))

Do you require more changes?